Sources/POSIX/Error.swift

```diff
@@ -38,7 +38,7 @@ import func libc.strerror_r
 extension SystemError: CustomStringConvertible {
     public var description: String {
         func strerror(_ errno: Int32) -> String {
-            for cap in [64, 128, 512, 1024, 2048, 4096, 8192, 16_384] {
+            for cap in sequence(first: 64, next: { $0 < 16 * 1024 ? $0 * 2 : nil }) {
```

Comment:
With Swift 3.1, the sequence variant won't need the ternary anymore with the introduction of prefix(while: )